src/library/vm/vm.cpp

```diff
@@ -3283,7 +3429,7 @@ void vm_state::run() {
                     goto main_loop;
                 }
             }
-            invoke_global(decl);
+            invoke(decl);
```

Comment:
I don't remember making this change. It might have found its way in by mistake. Do you see a point in keeping it?

Reply:
It would only matter if the override was a different kind of function, i.e. if you override a definition with a meta constant.

Reply:
I don't think that change would be appropriate because the information about cfun, external, etc, is stored in the `vm_decl` and `get_vm_decl` will return the `vm_decl` of the override so the right invocation strategy should be invoked.